src/lib/termdesc.c

```diff
@@ -802,6 +802,11 @@ int interrogate_terminfo(tinfo* ti, int fd, const char* termname, unsigned utf8,
         invertsixel = true;
       }
     }
+    if(qterm == TERMINAL_FOOT){
+      if(compare_versions(ti->termversion, "1.8.2") >= 0){
+        invertsixel = true;
```

Comment:
i'd rather put this in `apply_term_heuristics` where all this other kind of crap logic is. thanks a lot for the patch, though!

Reply:
actually, nevermind, this is where we do the mlterm detection and thus right where it ought go.